Assets/Scripts/Models/Functions/FunctionsManager.cs

```diff
@@ -112,6 +112,9 @@ public static void JobComplete_FurnitureBuilding(Job theJob)
     {
         WorldController.Instance.World.PlaceFurniture(theJob.JobObjectType, theJob.tile);
 
+        // Let our workspot tile know it is no longer reserved for us
+        WorldController.Instance.World.UnreserveTileAsWorkSpot((Furniture)theJob.buildablePrototype, theJob.tile);
```

Comment:
Oh. I see. Yes I was thinking that if you want multiple furnitures to share the same spot. It will fail at the Job stage when trying to build the same Furniture.